pkg/services/ngalert/store/org.go

```diff
@@ -0,0 +1,26 @@
+package store
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+)
+
+type OrgStore interface {
+	GetOrgs() ([]int64, error)
```

Comment:
`GetOrgs` could be `GetOrgs(context.Context)` so we don't have to use `context.Background()` on Line 15?

Reply:
Thanks! I have propagated the context to this query, but being honest this wouldn't be the egregious offender of this area. I have added an item to https://github.com/grafana/grafana/issues/37222 so that when I fix the N+1 query I can ensure the new query propagates the context.